apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -417,8 +417,6 @@ class ReviewTab extends Component {
           {codeReviewEnabled && !viewAsTeacher && (
             <>
               <ReviewNavigator
-                onSelectPeer={this.onSelectPeer}
-                onReturnToProject={this.onClickBackToProject}
```

Comment:
Can we remove these methods from `ReviewTab.jsx`?

Reply:
Whoops, yes I'll remove them